### PR TITLE
2.0-preview <- master

### LINKED ISSUE #402: [2.0] Error after update to latest code
### Description
After updating the code (Polymer and iron-list) to the latest version I see an error when the iron-lists items collection is rendered.

### Expected outcome

I would expect to see the list as before.

### Actual outcome
Uncaught TypeError: Cannot set property '__key__' of undefined
    at HTMLElement.<anonymous> (iron-list.html:1197)
    at HTMLElement._iterateItems (iron-list.html:1162)
    at HTMLElement._assignModels (iron-list.html:1192)
    at HTMLElement._update (iron-list.html:910)
    at HTMLElement._render (iron-list.html:1009)
    at HTMLElement.<anonymous> (iron-list.html:1432)
    at _timer._asyncModule.run (debounce.html:40)
    at web-animations-next-lite.min.js:16

### Live Demo

https://jsbin.com/sazojeqoci/2/edit?html,output

### Steps to reproduce

1. Put a `iron-list` element in the page.
2. Have a style for the list
3. Update the items array

### Browsers Affected

- [ x] Chrome

### PR DESCRIPTION
**Pull request by keanulee**
Fixes #402

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -76,6 +76,6 @@
     }
   },
   "resolutions": {
-    "test-fixture": "safari9-constructor"
+    "test-fixture": "^3.0.0-rc.1"
```
Comment (blasten) [inline bower.json]: does it still need this resolution?

**Comment by blasten** [comment]
minor comment. otherwise LGTM.

File: bower.json (hunk 1 above)
Comment (keanulee) [inline bower.json]: No, I'll remove it.

**Comment by blasten** [comment]
Let's look into that failing test in 2.0-preview.

**Comment by keanulee** [comment]
@blasten Wouldn't worry about that - the latest test on 2.0-preview is ✅  https://travis-ci.org/PolymerElements/iron-list/branches

**Pull request merged**